### PR TITLE
Implement Bookstore API

### PR DESCRIPTION
**Pull request by JuliCarracedo**
This PR by @JuliCarracedo includes the implementation of Bookstore API as a means of storage:

- Upload created books to API
- Remove from API the locally removed books
- On load, fill the store with the API list of books.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(thunk),
+);
+
+export default store;
```
Comment (Emmyn5600) [inline src/redux/configureStore.js:14]: - [x] Your project isn't running locally, kindly make sure you installed `redux-thunk` successfully.

rr27

rr28

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    case ADD_BOOK:
+      return ([...state, action.book]);
+    case REMOVE_BOOK:
+      return (state.filter((book) => book.item_id !== action.item_id));
+    default:
+      return state;
+  }
+};
+
+export default reducer;
```
Comment (Emmyn5600) [inline src/redux/books/books.js]: - [x] - [x] Good Job, however, let us modify our implementation a little bit, to keep our code clean and organized, let us create a new file called API.js to keep `(POST, DELETE and GET)` and book.js  file to keep the `action types, creator and reducers.`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

### Hi, @JuliCarracedo 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

   ###  Changes required :recycle: 
    
    -  Please check the comment under the review too

    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/redux/books/books.js (hunk 2 above)
Comment (henatan99) [reply src/redux/books/books.js]: This issue is well addressed. Nice job! :clap:

File: src/redux/configureStore.js (hunk 1 above)
Comment (henatan99) [reply src/redux/configureStore.js:14]: This issue is resolved. Great job! :+1: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @JuliCarracedo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**